tests/channels-search.bats

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test of rust impl of `flox search`
+#
+# bats file_tags=search 
+#
+# ---------------------------------------------------------------------------- #
+
+load test_support.bash
+
+# ---------------------------------------------------------------------------- #
+
+# Helpers for project based tests.
+
+project_setup() {
+  export PROJECT_DIR="${BATS_TEST_TMPDIR?}/test"
+  rm -rf "$PROJECT_DIR"
+  mkdir -p "$PROJECT_DIR"
+  pushd "$PROJECT_DIR" >/dev/null || return
+  PROJECT_REGISTRY="$BATS_TEST_DIR/search/registry.json"
+  export NIX_CONFIG="flake-registry = $PWD/registry.json"
```

Comment:
I guess this should also be on the todo list for the next PR?

Reply:
Fixed